lib/tessel/deployment/rust.js

```diff
@@ -1,21 +1,20 @@
 // System Objects
-var path = require('path');
 var http = require('http');
-var path = require('path');
 var url = require('url');
 
 // Third Party Dependencies
 var fs = require('fs-extra');
 var tags = require('common-tags');
 var tar = require('tar');
-var toml = require('toml');
 var Reader = require('fstream').Reader;
+var tags = require('common-tags');
```

Comment:
Now there are two `var tags = require('common-tags');`, so let's just clean up the whole section, change to: 

```js
// Third Party Dependencies
var fs = require('fs-extra');
var fstream = require('fstream');
var tags = require('common-tags');
var tar = require('tar');

var Reader = fstream.Reader;
```